markdown/about.md

```diff
@@ -17,7 +19,12 @@ The [Informatics Infrastructure team](/it) provides support for the production o
 - [Paul Davis](/it#paul-davis)
 - Beth Yates
 
-We welcome external collaboration on our projects - if you'd like to be involved, [drop us a message](https://github.com/sanger-tol/pipelines-website/issues/new?assignees=priyanka-surana%2Cmuffato&labels=connect&projects=&template=contact_us.yaml&title=%5BContact+Us%5D%3A+).
+## Collaborations
+
+We welcome external contributions and collaboration on our projects.
+Iif you'd like to be involved, [drop us a message](https://github.com/sanger-tol/pipelines-website/issues/new?assignees=priyanka-surana%2Cmuffato&labels=connect&projects=&template=contact_us.yaml&title=%5BContact+Us%5D%3A+) or comment directly on an existing GitHub issue.
+
+Please note that we don't have funded internships or student placements at the moment, but are still very happy to help and mentor.
```

Comment:
Do we really need this sentence?

Reply:
I thought it'd be useful to set the expectations, but maybe it just looks bad. Happy to remove it if you think it's not needed